### PR TITLE
update changelog

### PR DESCRIPTION
**Pull request by psychocoderHPC**
ci: no-compile

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,150 @@
 Changelog
 =========
 
+0.8.0
+-----
+
+**Date:** 2024-12-XX
+
+Key highlights for this release are a new atomic model FLYonPIC 2.0, the shadowgraphy plugin with advanced diagnostics for laser-plasma interactions,
```
Comment (BrianMarre) [inline CHANGELOG.md]:
```suggestion
Key highlights for this release are a new atomic physics model FLYonPIC 2.0, the shadowgraphy plugin with advanced diagnostics for laser-plasma interactions,
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,150 @@
...
+    - fix slow tbg substitution #4905
+    - fix growth rate and charge conservation test tools #4834
+ 
+**Documentation:**
+- PIC:
+    - replace cupla fully by alpaka #4838
+- PMacc:
+    - replace cupla fully by alpaka #4838
+- other:
+    - topic remove cuda11.3 and g++9/CUDA support #5010
```
Comment (BrianMarre) [inline CHANGELOG.md]:
```suggestion
    - remove cuda11.3 and g++9/CUDA support #5010
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,150 @@
...
+    - change class and function names in radPlugin to camelCase #4734
+    - add calorimeter python module #4697
+- tools:
+    - pre-commit #4792
+    - remove hemera k80 and k20 partition #4779
+    - add calorimeter python module #4697
+
+**Features:**
+- PIC:
+    - BSI minimal implementation #5205
```
Comment (BrianMarre) [inline CHANGELOG.md]:
```suggestion
```
this is a FLYoPIC specific feature and should not mentioned separately

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,150 @@
...
+    - change pulse laguerre mode definition #4964
+    - remove broken B field implementation of dispersive laser #4932
+    - new flexible kernel for particle creation #4918
+    - add particle sparser manipulator #4916
+    - implementation of Maxwell Juettner distribution #4826
+    - add feature `--dump-metadata` #4812
+    - add openPMD-viewer test #4747
+    - PIConGPU unit tests #4723
+    - add CKC Solver #4661
+    - FLYonPIC v2.0 #4551
```
Comment (BrianMarre) [inline CHANGELOG.md]:
```suggestion
    - FLYonPIC v2.0, with energy conserving ADK and BSI field ionization #4551
```

**Review by BrianMarre — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (steindev) [reply CHANGELOG.md]: Also, 'the shadowgraphy plugin as advanced synthetic diagnostic of laser plasma interactions.'

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,150 @@
...
+enhanced OpenPMD functionality, and support for the RISC-V ecosystem.
+As usual, this release features fixes of PIConGPU and extensions and clarifications of the documentation.
+This is the last release with C++17 support before we switch to C++20.
+
+**Bug Fixes:**
+- PIC:
+    - fix C++20 compile #5015
+    - fix Gaussian and Dispersive pulse #4889
+- PMacc:
+  - Fix null parameter in MPI_Init_thread #5230
```
Comment (steindev) [inline CHANGELOG.md]: increase indent.

File: CHANGELOG.md (hunk 3 above)
Comment (psychocoderHPC) [reply CHANGELOG.md]: added to the flyonpic point

**Comment by psychocoderHPC** [comment]
@steindev I fixed all mentioned parts

**Comment by psychocoderHPC** [comment]
@steindev I think we missed to mention the new laser profiles. COuld you please provide a sentence for the introduction.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,149 @@
...
+- plugins:
+    - optimize OpenPMD IO #4882
+    - openPMD: add parameter `particleIOChunkSize` #4874
+    - bump openPMD-api to minimal version 0.15 #4782
+    - simplify output structure colorimeter plugin  #4695
+    - transitionRadiation plugin openPMD output #4649
+- tools:
+    - refactor `pic-build` and `pic-configure` #4936
+    - Pre-commit #4792
+
```
Comment (ikbuibui) [inline CHANGELOG.md:142]:
```suggestion
- PICMI:
    - Added support to configure the moving window #4987
    - Added support for custom user inputs #4996
    - Added the ability to configure ionization model interactions #5007
    
```

Comment (ikbuibui) [reply CHANGELOG.md:142]: @BrianMarre does this look okay?

Comment (BrianMarre) [reply CHANGELOG.md:142]: looks good

**Comment by steindev** [comment]
@psychocoderHPC:
'..., the shadowgraphy plugin as advanced synthetic diagnostic of laser plasma interactions, the new laser profile FromOpenPMDPulse allows to load electric fields in transverse space and time domain into the simulation via incidentField, ...'

~But please double check the name of the laser profile and incidentField method (=the name of the `.param` file). I am on phone and cannot really do this now.~ Done

Thanks!

**Review by BrianMarre — APPROVED** [review]

File: CHANGELOG.md (hunk 6 above)
Comment (psychocoderHPC) [reply CHANGELOG.md:142]: I added this to the changelog, if PICMI needs it'S own section someone must update the python changelog generator tool to support automatic changelog creation for PICMI

**Comment by psychocoderHPC** [comment]
@ComputationalRadiationPhysics/picongpu-developers Last chance to check the changelog, the release date is already set for tomorrow.

**Review by steindev — APPROVED** [review]

**Pull request merged**